src/app/home/avatar-generator/selector/atribute-selector.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import {AvatarDecoratorItem, AvatarOptions} from '../Avatartypes';
+import {buildAvatar} from '../AvatarUtils';
+import {getPreview} from '../AvatarSelectorPath';
+
```

Comment:
Please add the "component" suffix to the name of this file: attribute-selector.component.ts

Reply:
fixed